src/SFA.DAS.Commitments.Application/Commands/UpdateApprenticeshipStatus/StopApprenticeshipCommandHandler.cs

```diff
@@ -163,13 +163,14 @@ private void ValidateChangeDateForStop(DateTime dateOfChange, Apprenticeship app
                 if ( dateOfChange.Date < apprenticeship.StartDate.Value.Date)
                     throw new ValidationException("Invalid Date of Change. Date cannot be before the training start date.");
 
-                if ( apprenticeship.PaymentStatus != PaymentStatus.PendingApproval && 
-                    _academicYearValidator.Validate(dateOfChange.Date) == AcademicYearValidationResult.NotWithinFundingPeriod)
+                if (apprenticeship.PaymentStatus != PaymentStatus.Withdrawn)
                 {
-                    throw new ValidationException("Invalid Date of Change. Date cannot be before the academic year start date.");
-                }
-                    
-                   
+                    if (apprenticeship.PaymentStatus != PaymentStatus.PendingApproval &&
```

Comment:
Might as well combine this `if` and the enclosing `if` into one.

Reply:
I was going to but I thought it more readable to keep them separate.

Reply:
It's only a style thing.